src/authkit-callback-route.ts

```diff
@@ -57,6 +69,16 @@ export function authLoader(options: HandleAuthOptions = {}) {
         session.set('jwt', encryptedSession);
         const cookie = await commitSession(session);
 
+        if (onSuccess) {
+          await onSuccess({
+            accessToken,
+            impersonator: impersonator ?? null,
+            oauthTokens: oauthTokens ?? null,
+            refreshToken,
```

Comment:
This feels potentially dangerous. In other libraries we never provide the refresh token unencrypted like this, instead extracting it later from the sealed session. This library unfortunately lacks a "refreshSession" like the Next.js library has (which extracts the refresh token from the encrypted session via the cookie password on the server), but it still feels unsafe to provide this here.

Reply:
Could you say a bit more about why it feels unsafe? Here's my thinking about why it is not:

* The `refreshToken` (or other parameters, like `oauthTokens`) are only visible to the application's server-side code. I'm certainly no Remix expert, so let me know if this is possibly not true for this (or other) "loaders".
* Whether the token is provided to the application after being extracted from either, the `encryptedSession`, or the response of a server-side API call to `/user_management/authenticate` (like in this case), the value is plaintext.

Reply:
It comes from me also not being a Remix expert and not knowing if what's in this function can be leaked to the client.

An alternative would be to not provide the refresh token here and provide a `refreshSession` method like with authkit-nextjs instead. I think abstracting away the refresh logic is probably better DX, plus if we return the refresh token now then we'll have to support that forever.

Reply:
Alright, I can pare down the data given to the `onSuccess` callback to just the data we think a developer might want currently for side-effects. I think that'll probably be `oauthTokens`, which was the instigator, and maybe also `user` and `impersonator`.

We can leave `refreshToken` and `accessToken` out of it since those are managed by other parts of the SDK. WDYT?

Reply:
And in case we have any more doubt about [where `loader` is run`](https://remix.run/docs/ja/main/route/loader#loader), and whether this callback could accidentally leak data to the client:

> This function is only ever run on the server.

> You need to return a [Fetch Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) from your loader.

This callback must return `void` and therefore cannot affect what is returned to the client. The exception might be if an error is thrown, but that seems any more likely to leak data than any other `loader` in the developer's application.